pkg/resources/resources.go

```diff
@@ -635,6 +635,14 @@ alertmanager_config: |
   receivers:
     - name: 'null'
 `
+
+	// Konnectivity
+	KonnectivityDeploymentName         = "konnectivity-agent"
+	UserClusterKonnectivityDaemonSetName          = "konnectivity-agent"
+	UserClusterKonnectivityClusterRoleBindingName = "system:konnectivity-server"
+	UserClusterKonnectivityServiceAccountName     = "konnectivity-agent"
+	KonnectivityAgentContainer                    = "konnectivity-agent"
+	KonnectivityProxyServiceName                  = "konnectivity-server"
```

Comment:
Just from looking at this I am not sure if any of these names are used inside the usercluster. If so, please add some prefix so we do not actually clash names with whatever the user wants to do in their cluster. Also please do not use "kubermatic" or "kkp" as the prefix, as we _try_ to be somewhat white-label.

If the names are only used in the controlplane (i.e. the `cluster-....` namespace on the seed), then the names are totally fine.

Reply:
Some of them are used in the user cluster. Didn't think about prefixing them. Thanks for the suggestion. :pray:

Reply:
> Also please do not use "kubermatic" or "kkp" as the prefix, as we try to be somewhat white-label.

@xrstf This is a good point, now I am thinking maybe we should also drop the `kubermatic` prefix for user cluster MLA resources, currently, those resources are prefixed with `system:kubermatic:mla`, we can just have `system:mla`.

Reply:
> Also please do not use "kubermatic" or "kkp" as the prefix, as we try to be somewhat white-label.

@xrstf This is a good point, now I am thinking maybe we should also drop the `kubermatic` prefix for user cluster MLA resources, currently, those resources are prefixed with `system:kubermatic:mla`, we can just have `system:mla`.